spec/features/flows/sp_authentication_flows_spec.rb

```diff
@@ -381,4 +381,12 @@ def submit!
       end
     end
   end
+
+  def complete_phone_form_with_valid_phone
+    phone = Faker::PhoneNumber.cell_phone
+    until PhonyRails.plausible_number? phone, country_code: :us
+      phone = Faker::PhoneNumber.cell_phone
+    end
+    fill_in 'user_phone_form_phone', with: phone
```

Comment:
I had meant just pulling the part that generates the valid-looking phone number into a helper, not the fillin in part, but this works too